src/opnsense/mvc/app/models/OPNsense/Trust/Trust.xml

```diff
@@ -0,0 +1,81 @@
+<model>
+    <mount>//OPNsense/Trust</mount>
+    <version>1.0.0</version>
+    <description>Trust/certificate manage</description>
+    <items>
+        <cas>
+            <ca type="ArrayField">
+                <refid type="TextField">
+                </refid>
+                <cauuid type="TextField">
+                </cauuid>
+                <descr type="TextField">
+                </descr>
+                <crt type="TextField">
+                </crt>
+                <prv type="TextField">
+                </prv>
+                <serial type="IntegerField">
+                </serial>
+            </ca>
+        </cas>
+        <certs>
+            <cert type="ArrayField">
+                <refid type="TextField">
+                </refid>
+                <cauuid type="TextField">
+                </cauuid>
```

Comment:
this should probably be a `ModelRelationField`

Reply:
In my opinion, even better to use CertificateField. I tried to use CertificateField, but I'm having problems with the migration. The certificates are not migrated. Does not pass the validator "InclusionIn". It checks the list of available CA, but they have just been added and the validator does not know them. In all other cases CertificateField works great. ModelRelationField also uses the validator "InclusionIn", most likely, will have the same problems with migration.

I will think about how you can use CertificateField and use it with migration.

Reply:
@fichtner @AdSchellevis looks like a problem with the framework